### PR TITLE
final updates for onetrust before going public

### PR DESCRIPTION
**Pull request by tcgilbert**
### Proposed changes

Updating event and event properties tables

### Merge timing
asap

### CODE REVIEW

File: src/connections/sources/catalog/cloud-apps/onetrust/index.md [hunk 1]
```diff
@@ -30,14 +29,22 @@ OneTrust allows you to configure the userId from various sources from within the
 
 ## Events
 
-Use the OneTrust integration to define the events to be sent to Segment These events can be tailored to align with your existing process or particular use case.
+The following table lists events that OneTrust sends to Segment. These events show up as tables in your Warehouse and as regular events in your other Destinations.
+
+|  Event Name | Description |
+|  ------ | ------ |
+| Consent Updated | User has given or withdrawn consent for an express purpose |
```
Comment (stayseesong) [inline src/connections/sources/catalog/cloud-apps/onetrust/index.md:36]:
```suggestion
| Consent Updated | The user gave or withdrew consent for an express purpose. |
```

File: src/connections/sources/catalog/cloud-apps/onetrust/index.md [hunk 2]
```diff
@@ -30,14 +29,22 @@ OneTrust allows you to configure the userId from various sources from within the
...
 
 ## Event Properties
 
-The OneTrust integration allows you to define event properties within the following constraints:
+The following table lists event properties included with all events Segment receives from Braze.
+
+|  Property Name | Type | Description |
+|  ------ | ------ | ------ |
+| Purpose ID | string | GUID assigned to the purpose in OneTrust |
+| Purpose Name | string | Name given to the purpose in OneTrust for which the user is giving or withdrawing their consent |
```
Comment (stayseesong) [inline src/connections/sources/catalog/cloud-apps/onetrust/index.md:45]:
```suggestion
| Purpose Name | string | The name given to the purpose in OneTrust for which the user is gives or withdraws their consent. |
```

File: src/connections/sources/catalog/cloud-apps/onetrust/index.md [hunk 3]
```diff
@@ -30,14 +29,22 @@ OneTrust allows you to configure the userId from various sources from within the
...
+| Consent Updated | User has given or withdrawn consent for an express purpose |
 
 ## Event Properties
 
-The OneTrust integration allows you to define event properties within the following constraints:
+The following table lists event properties included with all events Segment receives from Braze.
+
+|  Property Name | Type | Description |
+|  ------ | ------ | ------ |
+| Purpose ID | string | GUID assigned to the purpose in OneTrust |
```
Comment (stayseesong) [inline src/connections/sources/catalog/cloud-apps/onetrust/index.md:44]:
```suggestion
| Purpose ID | string | The GUID assigned to the purpose in OneTrust. |
```

File: src/connections/sources/catalog/cloud-apps/onetrust/index.md (hunk 2 above)
Comment (stayseesong) [inline src/connections/sources/catalog/cloud-apps/onetrust/index.md:45]:
```suggestion
| Purpose Name | string | The name given to the purpose in OneTrust for which the user gives or withdraws their consent. |
```

File: src/connections/sources/catalog/cloud-apps/onetrust/index.md [hunk 4]
```diff
@@ -30,14 +29,22 @@ OneTrust allows you to configure the userId from various sources from within the
...
+| Purpose Name | string | Name given to the purpose in OneTrust for which the user is giving or withdrawing their consent |
+| Status | string | Current status of the users consent after the update. Options include ACTIVE, WITHDRAWN, PENDING, OPT_OUT, EXPIRED, NO_CONSENT, HARD_OPT_OUT, and NOT_OPTED_OUT |
```
Comment (stayseesong) [inline src/connections/sources/catalog/cloud-apps/onetrust/index.md:46]:
```suggestion
| Status | string | The current status of the users consent after the update. Options include ACTIVE, WITHDRAWN, PENDING, OPT_OUT, EXPIRED, NO_CONSENT, HARD_OPT_OUT, and NOT_OPTED_OUT. |
```

**Review by stayseesong — CHANGES_REQUESTED** [review]

**Comment by stayseesong** [comment]
@tcgilbert when is this source going public so that I know when to merge

**Review by stayseesong — APPROVED** [review]

**Comment by tcgilbert** [comment]
Hi @stayseesong, I'll make it live right now

**Pull request merged**